appveyor.yml

```diff
@@ -1,25 +1,24 @@
 version: 0.2.{build}
 environment:
   TARGET: x86_64-pc-windows-msvc
+  RUST_BACKTRACE: 1
   matrix:
     - PYTHON: "C:/Python35-x64"
     - PYTHON: "C:/Python36-x64"
```

Comment:
what about adding `x32` versions and Python37? this can be done in a separate PR though

Reply:
Yeah, I think we should add Python37 but I'm not sure we should support 32bit architecture.
And just a nit, but there's no architecture named 'x32'.
The 32bit version of 'x86-64' is called 'x86'.

Reply:
Thanks, I'll keep it in mind.